_ext/jboss_developer.rb

```diff
@@ -387,10 +387,11 @@ def exists?(page)
       #
       # Returns the Faraday::Response
       def update_page(page, content)
-        payload = create_payload page, content
         path = create_path page
+        puts "Updating page '#{@base_url}#{path}' with content '#{page.output_path}'"
```

Comment:
If we want to keep it in, I suggest we do some logging here instead of simply `puts`